woocommerce/woocommerce
# Update playwright api-core-tests to associate orders with real products to prevent extension issues for those that validate product ids

### All Submissions:






### Changes proposed in this Pull Request:

Update playwright api-core-tests to associate orders with real products to prevent extension issues for those that validate product ids

Closes https://github.com/woocommerce/woocommerce/issues/37220.



### How to test the changes in this Pull Request:




1. Run extensions test in QIT
2. Observe test results
3. Tests should now pass (previously they were failing on extensions that were validating against product ids)



### Other information:




### FOR PR REVIEWER ONLY:
